lib/betagouvbot/accountrequest.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module BetaGouvBot
+  class AccountAction
+    attr_accessor :name
+    attr_accessor :password
+
+    def initialize(name, password)
+      @name = name
+      @password = password
+    end
+
+    def execute
```

Comment:
Thinking back on our IRL conversation earlier, I see a fundamental difference between `execute` (as used in Action objects) and `call` (as used in the top-level modules webhooks call into):
 - `call` takes parameters
 - `execute` is nullary (an Action is basically a method object)

It makes sense to me to keep the names distinct on this basis ?